src/trading/ClaimProceeds.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+import 'ROOT/reporting/Interfaces.sol';
+import 'ROOT/extensions/MarketFeeCalculator.sol';
+
+/**
+ * @title ClaimProceeds
+ * @dev This allows users to claim their money from a market by exchanging their shares 
+ */
+contract ClaimProceeds is Controlled {
+	uint256 private constant THREE_DAYS = 3 * 24 * 60 * 60;
+
+	function publicClaimProceeds(IMarket market) onlyInGoodTimes public returns(uint256) {
```

Comment:
This function should return a `bool`.  As mentioned above, this was returning an int in Serpent only because there is no boolean type in Serpent.